quaint/src/connector/metrics.rs

```diff
@@ -3,12 +3,7 @@ use tracing::{info_span, Instrument};
 use crate::ast::{Params, Value};
 use std::{future::Future, time::Instant};
 
-pub(crate) async fn query<'a, F, T, U>(
-    tag: &'static str,
-    query: &'a str,
-    params: &'a [Value<'_>],
-    f: F,
-) -> crate::Result<T>
+pub async fn query<'a, F, T, U>(tag: &'static str, query: &'a str, params: &'a [Value<'_>], f: F) -> crate::Result<T>
```

Comment:
I'm not a fan of having this public, but it's the right call, this is the smallest change that works and we're past trying to keep the quaint public API clean.